pkg/controller/populators/populator-base.go

```diff
@@ -220,6 +213,31 @@ func (r *ReconcilerBase) createPVCPrime(pvc *corev1.PersistentVolumeClaim, sourc
 	return pvcPrime, nil
 }
 
+type updatePVCAnnotationsFunc func(pvc, pvcPrime *corev1.PersistentVolumeClaim)
+
+var desiredAnnotations = []string{cc.AnnPodPhase, cc.AnnPodReady, cc.AnnPodRestarts, cc.AnnPreallocationRequested, cc.AnnPreallocationApplied, cc.AnnRunningCondition, cc.AnnRunningConditionMessage, cc.AnnRunningConditionReason, cc.AnnBoundCondition, cc.AnnBoundConditionMessage, cc.AnnBoundConditionReason}
```

Comment:
If I understand this correctly, the dv will have bound condition true if pvc' is bound.  That does not seem right to me

Reply:
the update boundcondition on the dv doesnt update the condition to true unless the pvc is bound: 
https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/controller/datavolume/conditions.go#L128-L143
so seems to me its ok.

Reply:
Why bother passing over the annotations then?

Reply:
I remember there was some tests checking this. I can remove this update and see how it affects our tests if it bothers you.